llvm-runtimes/libatomic-stub/libatomic-stub-1.0.0.ebuild

```
@@ -0,0 +1,23 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Stub library which allows compiler-rt to be used as an libatomic"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+S="${WORKDIR}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	llvm-runtimes/compiler-rt[atomic-builtins(-)]
+	!sys-devel/gcc
+"
+
+src_install() {
+	local libdir="${ED}/usr/$(get_libdir)"
+	mkdir -p "${libdir}" || die
+	${AR} rc "${libdir}/libatomic.a" || die
```

```suggestion
	${AR} rc libatomic.a || die
	dolib libatomic.a
```

Also, I suppose this should be multilib?